GrandCentralBoard/Widgets/Harvest/HarvestWidget.swift

```diff
@@ -10,22 +10,35 @@ import GrandCentralBoardCore
 
 
 final class HarvestWidget : Widget {
-    let view: UIView
+
+    private let widgetView: AreaBarChartView
+
     let sources: [UpdatingSource]
 
-    init(view: HarvestWidgetView, sources: [UpdatingSource]) {
-        self.view = view
+    var view: UIView {
+        return widgetView
+    }
+
+    init(view: AreaBarChartView, sources: [UpdatingSource]) {
+        self.widgetView = view
         self.sources = sources
     }
 
     func update(source: UpdatingSource) {
         guard let source = source as? HarvestSource else { return }
 
         source.read({ result in
-            self.updateView(result)
+            self.updateViewWithViewModel(result)
         })
     }
 
-    func updateView(result: HarvestSource.ResultType) {
+    func updateViewWithViewModel(viewModel: HarvestSource.ResultType) {
```

Comment:
The names used here suggests that the argument is the view model, while it's just a result